actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkUploadBackgroundJobActor.java

```diff
@@ -225,14 +251,16 @@ private void callUpdateUser(User user, BulkUploadProcessTask task, String orgNam
     }
   }
 
-  private Map<String, Object> getOrg(String orgId) {
-    Map<String, Object> result =
-        ElasticSearchUtil.getDataByIdentifier(
-            ProjectUtil.EsIndex.sunbird.getIndexName(),
-            ProjectUtil.EsType.organisation.getTypeName(),
-            orgId);
-    if (result != null && result.size() > 0) {
-      return result;
+  private Organisation getOrgDetails(Map<String, Object> userMap) {
+    if (StringUtils.isNotBlank((String) userMap.get(JsonKey.EXTERNAL_ID))) {
+      Map<String, Object> filters = new HashMap<>();
+      filters.put(JsonKey.EXTERNAL_ID, userMap.get(JsonKey.EXTERNAL_ID));
+      if (CollectionUtils.isNotEmpty(organisationClient.esSearchOrgByFilter(filters))) {
+        return organisationClient.esSearchOrgByFilter(filters).get(0);
```

Comment:
Can this return empty list? We are always reading first element.

Reply:
yes , we are checking for not empty

Reply:
ok